services/web/server/src/simcore_service_webserver/scicrunch/_config.py

```diff
@@ -6,15 +6,16 @@
 
 class SciCrunchSettings(BaseSettings):
```

Comment:
Can you inherit from settings-library here? Like this the case_sensitive below option is removed.

Reply:
Good point, but the problem is that the new settings-library is still not included in the web server. I am doing that in a separate pull request